guides/updating/update-indy-sdk-to-askar.md

```diff
@@ -99,16 +99,21 @@ The agent is not allowed to be initialized to run this script. This makes sure n
 :::
 
 ```typescript
-import { IndySdkToAskarMigrationUpdater } from '@hyperledger/indy-sdk-to-askar-migration'
 import { agentDependencies } from '@aries-framework/react-native'
 import { AskarModule } from '@aries-framework/askar'
+import { IndySdkToAskarMigrationUpdater } from '@aries-framework/indy-sdk-to-askar-migration'
+import { ariesAskar } from '@hyperledger/aries-askar-react-native'
 
 const oldAgent = new Agent({
   config: {
     /* ... */
   },
   modules: {
-    modules: { askar: new AskarModule() },
+    modules: { 
```

Comment:
I think there was a duplicated `modules` key here

Reply:
Ah, good point. I totally overlooked that one :)